clients/adapter-amqp/src/test/java/org/eclipse/hono/adapter/client/amqp/RequestResponseClientTest.java

```diff
@@ -0,0 +1,379 @@
+/*******************************************************************************
+ * Copyright (c) 2016, 2020 Contributors to the Eclipse Foundation
```

Comment:
2020

Reply:
I had deliberately kept the origin year because most of the code has been copied over from the legacy client.